src/main/java/edu/wpi/first/pathweaver/Field.java

```diff
@@ -101,6 +107,11 @@ private void setRealWidth(Quantity<Length> width) {
 		return rLength;
 	}
 
+	public double getDefaultLength() {
+		UnitConverter toFeet = getUnit().getConverterTo(PathUnits.FOOT);
+		return toFeet.convert(realLength() * 0.25);
+	}
```

Comment:
I am not sure if these methods should live in the Field class. What do you think about moving the implementation into WpilibPath by where we need to use the info?

Reply:
Is there a good way to do that?  Because this is in the constructor body I don't think I can call other methods in WpilibPath.  Could we restructure this...yeah probably but that seemed like it would touch a lot of other things.  And either way it is kind of an arbitrary helper and to me there wasn't a clear place for it.

Reply:
It is safe to call private static methods in a constructor. Can you make it a static method that accepts a field and outputs the width?

Reply:
yep thanks!